Release From Develop Branch into Main v0.2.0 (#57)
cmpsc-481-s22-m1/PyPiCounter: README.md

## Getting Started

### Importing into Another Grading Tool

> 1. Install from PyPi by running `poetry add pypicount` in your terminal.

Reviewer: Should be the name of the package on PyPI:
```suggestion
1. Install from PyPi by running `poetry add pypi-counter` in your terminal.
```

> 2. Import the PyPiCounter package withe the syntax: `import pypicount`.

Reviewer: Should be the package name specified in your `pyproject.toml`:
```suggestion
2. Import the PyPiCounter package with the syntax: `import pypi_count`.
```

# returns the number of import statements

count_for_loops(file_name)                           
# returns the number of for loops

count_while_loops(file_name)                           
# returns the number of while loops

count_function_definitions(file_name)            
# returns the number of function definitions  

count_functions_without_docstrings(file_name)         
# returns the number of functions without docstrings

count_functions_with_docstrings(file_name)          
# returns the number of functions with docstrings

count_classes_with_docstrings(file_name)                  
# returns the number of classes with docstrings

count_classes_without_docstrings(file_name)             
# returns the number of classes without docstrings

count_function_parameters(file_name, function_name)    
# returns the number of function parameters
# after specifying the function name

count_assignment_statements(file_name)              
# returns the number of assignment statements

```

### Stand-Alone Usage

Users can get started with this project by following the following steps:

1. Install the package with pip, using the command `pip install pypicount`

> or `pipx install pypicount`

Reviewer: ```suggestion
or `pipx install pypi-counter`
```

# returns the number of import statements

count_for_loops(file_name)                                   
# returns the number of for loops

count_while_loops(file_name)                                 
# returns the number of while loops

count_function_definitions(file_name)                        
# returns the number of function definitions  

count_functions_without_docstrings(file_name)                
# returns the number of functions without docstrings

count_functions_with_docstrings(file_name)                   
# returns the number of functions with docstrings

count_classes_with_docstrings(file_name)                     
# returns the number of classes with docstrings

count_classes_without_docstrings(file_name)                  
# returns the number of classes without docstrings

count_function_parameters(file_name, function_name)          
# returns the number of function parameters
# after specifying the function name

count_assignment_statements(file_name)                       
# returns the number of assignment statements

```

### Stand-Alone Usage

Users can get started with this project by following the following steps:

> 1. Clone this repository and `cd` into the project folder

Reviewer 2: Instead of requesting that users clone the repository, since your tool is now on PyPI they can install it with `pipx` or `pip`, which you should recommend. You can move the `poetry install`-related suggestions to a "contributing" section, or remove them altogether.
